src/main.js

```diff
@@ -0,0 +1,73 @@
+class Numberdrome {
+  constructor() {
+    this.numbersList = [];
+  }
+
+  addNumber(n) {
+    if (typeof n !== 'number' || !isFinite(n)) {
+      return;
+    }
+
+    this.numbersList.push(n);
+  }
+
+  removeNumber(n) {
+    this.numbersList.splice(this.numbersList.indexOf(n), 1);
+  }
+
+  sum() {
+    let sum = 0;
+    if (this.numbersList.length) {
+      this.numbersList.forEach(num => sum += num);
+    }
+
+    return sum;
+  }
+
+  product() {
+    let product = 1;
+    if (this.numbersList.length) {
+      this.numbersList.forEach(num => product *= num);
+    }
+
+    return product;
+  }
+
+  min() {
+    try {
+      if (!this.numbersList.length) {
+        throw new Error("Incorrect data: can`t find minimum of empty array");
+      };
+
+      return Math.min(...this.numbersList);
+    } catch(error) {
+      return error.message;
+    }
+  }
+
+  max() {
+    try {
+      if (!this.numbersList.length) {
+        throw new Error("Incorrect data: can`t find maximum of empty array");
+      };
+
+      return Math.max(...this.numbersList);
+    } catch(error) {
+      return error.message;
```

Comment:
you need to `throw an error` instead of `return message`

Reply:
Done